### ISSUE TITLE
Better navigation for extras sidebar

### ISSUE THREAD

**Issue by jerzakm** [threlte/threlte#708]
This is madness.

![image](https://github.com/threlte/threlte/assets/16734228/e6b13464-eeef-4575-9cf5-737ec25d9ac0)

Any suggestions on how to improve it?

1. /extras starting/overview page that is a grid with tiny screenshots and component names?

2. Sub categories? For this I currently see these as possibilities:
- Audio
- Camera & controls
- Shadows
- Scene/Environment(?) (Environment, Sky, Grid)
- Utils (Portal, Suspense)
- Others - everything else would go to others, and then once a certain category grows we make a new one and move these out.

**Comment by michealparks** [comment]
Catagories + alpha sorting within categories would make sense to me!

**Comment by ixxie** [comment]
I agree, categories and alphabetical sorting is good. I think it would also let us see opportunities for eventually breaking up extras into distinct packages if it gets too big.

**Comment by grischaerbe** [comment]
Done, I guess?
